Permanent/ViewControllers/FileDetailsViewController.swift

```diff
@@ -387,8 +389,16 @@ extension FileDetailsViewController: UICollectionViewDataSource {
             return "Original File Type".localized()
         }
     }
+    
+    func getAddressString() -> String {
```

Comment:
this method should be in the ViewModel, so that it's accessible from both this VC and the LocationVC

Reply:
Added method getAddressString in FilePreviewViewModel.